docs/faq.md

```diff
@@ -17,6 +17,12 @@
   Teams with a proven track record may apply for larger grants through [this application form](https://docs.google.com/forms/d/e/1FAIpQLSfMfjiRmDQDRk-4OhNASM6BAKii7rz_B1jWtbCPkUh6N7M2ww/viewform).
 </details>
 
+<details>
+  <summary><b>What activities/positions do you fund?</b></summary>
+
+  The Web3 Foundation's Grants Program aims to fund coding and research activities that are beneficial for the ecosystem as a whole. As such, we don't usually fund tangential costs, such as deployment and hosting costs, external consultants, maintenance or audits.
```

Comment:
For infrastruc we would fund audits and maintenance. Just not for projects that have a business model.

Reply:
And regarding external consultants, I guess it depends on what the external consultants actually do.

Reply:
But we don’t fund marketing or similar activities etc.

Reply:
I thought we didn't fund infrastructure?

Reply:
> And regarding external consultants, I guess it depends on what the external consultants actually do.

Can you elaborate? Coders are fine, I suppose? "business-specific activities" perhaps?

Reply:
> For infrastruc we would fund audits and maintenance. Just not for projects that have a business model.

How can we phrase that? `not-for-profit projects`? `public good projects`?

Reply:
> I thought we didn't fund infrastructure?

We fund key infrastructure. I’m referring to api clients, the go implementation, etc.

Reply:
> > And regarding external consultants, I guess it depends on what the external consultants actually do.
> 
> Can you elaborate? Coders are fine, I suppose? "business-specific activities" perhaps?

Yes, I was referring to code is fine ;-)

Reply:
> > For infrastruc we would fund audits and maintenance. Just not for projects that have a business model.
> 
> How can we phrase that? `not-for-profit projects`? `public good projects`?

Sounds good.